Resources/Audio/StationEvents/attributions.yml

```diff
@@ -11,4 +11,9 @@
 - files: ["sound_station_14.ogg"]
   license: "CC-BY-3.0"
   copyright: "Created by Donchan"
-  source: "https://www.youtube.com/watch?v=4nUpeYLKUns"
\ No newline at end of file
+  source: "https://www.youtube.com/watch?v=4nUpeYLKUns"
+
+- files: ["nukemass.ogg"]
+  license: "CC-BY-3.0"
+  copyright: "Created by Mrjajkes"
```

Comment:
```suggestion
  copyright: "Created by mrjajkes"
```
I asked mrjajkes for confirmation that this is the correct licence. But could you fix the name in the attribution like they requested?
![grafik](https://github.com/user-attachments/assets/71a654f3-e07c-4e31-80c8-ab0edbe161d2)